src/main/java/net/backupcup/mcde/screen/RunicTableScreen.java

```diff
@@ -107,14 +110,15 @@ public boolean mouseClicked(double mouseX, double mouseY, int button) {
 
             if (opened.isPresent() && opened.get() == slot.getSlot()) {
                 for (var choice : slot.choices()) {
-                    if (slotsRenderer.isInChoiceBounds(slot.getSlot(), choice.getSlot(), (int) mouseX, (int) mouseY))
+                    if (slotsRenderer.isInChoiceBounds(slot.getSlot(), choice.getChoiceSlot(), (int) mouseX, (int) mouseY)) {
```

Comment:
`getChoiceSlot` presents a confusing arrangement of words. Consider `getSlotChoice` if you are trying to reference the slot that is chosen by the user. If you are trying to refer to the slot, as an object, consider `getRunicTableSlot` to alleviate potential confusion.

Reply:
I defined choice as an inner slot for an enchantment (you see the `choices` when a player clicks on an empty slot) in the entirety of my codebase. So this change adds more clarity. I think the confusion could be from the fact that I use `getSlot` for both getting an object (`EnchantmentSlot` instance) and an enum (`Slots`), which could be described as a slot _position_. So I could rename all methods that return an enum to something like `getPosition`.

Reply:
Ah, this is a good point. I agree with your proposed change